src/Canvas.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+
+class Canvas extends Component {
+  constructor(props) {
+    super(props);
+    this.gameBoard = React.createRef();
+  }
+
+  componentDidUpdate() {
+    const context = this.gameBoard.current.getContext("2d");
+    this.drawGameState(context);
+  }
+
+  drawBall = (context) => {
+    context.beginPath();
+    context.arc(this.props.pongBall[0], this.props.pongBall[1], 10, 0, 360);
+    context.fillStyle = "red";
+    context.fill();
+    context.stroke();
+    context.closePath();
+  };
+
+  drawPaddle = (playerPaddle, context) => {
+    context.beginPath();
+    context.rect(playerPaddle[0], playerPaddle[1], 30, 10);
+    context.fillStyle = "black";
+    context.fill();
+    context.stroke();
+    context.closePath();
+  };
+
+  drawGameBoard = (context) => {
+    context.beginPath();
+    context.moveTo(0, 250);
```

Comment:
define the game board dimensions outside of this for easier editing. What if I want to toss this bad boy up on Time Square? :)